_unused_topics/monitoring-configuring-etcd-monitoring.adoc

```diff
@@ -39,7 +39,7 @@ $ oc -n openshift-monitoring edit configmap cluster-monitoring-config
 
 . Under `config.yaml: |+`, add the `etcd` section.
 +
-.. If you run `etcd` in static pods on your master nodes, you can specify the `etcd` nodes using the selector:
+.. If you run `etcd` in static pods on your control plane nodes (also known as the master nodes), you can specify the `etcd` nodes using the selector:
```

Comment:
I don't think that the "the" in the parentheses is actually appropriate in all cases, like how it's being used here. I'd say that at least in this particular case, you should remove the "the" completely. But it might be different for other circumstances.

Reply:
```suggestion
.. If you run `etcd` in static pods on your control plane nodes (also known as master nodes), you can specify the `etcd` nodes using the selector:
```